### DISCUSSION TITLE
Documentation mistake

### DISCUSSION THREAD

**Discussion by Fillox112** [betfinio/public#66] [Bugs and Reports]
In the affiliate intro on GitHub it says that 54% of all bets are dedicated for affiliates instead of 49%
![Snímek obrazovky 2024-10-14 v 15 08 23](https://github.com/user-attachments/assets/aae9ae3a-7994-4c62-a05f-71c5e15526dd)
.

**Comment by bf-tom** [comment]
fixed. thanks for reporting!
